python/grass/tools/support.py

```diff
@@ -0,0 +1,252 @@
+##############################################################################
+# AUTHOR(S): Vaclav Petras <wenzeslaus gmail com>
+#
+# PURPOSE:   API to call GRASS tools (modules) as Python functions
+#
+# COPYRIGHT: (C) 2025 Vaclav Petras and the GRASS Development Team
+#
+#            This program is free software under the GNU General Public
+#            License (>=v2). Read the file COPYING that comes with GRASS
+#            for details.
+##############################################################################
+
+"""
+The module provides support functionality for calling GRASS tools (modules)
+as Python functions.
+
+Do not use this module directly unless you are developing GRASS or its wrappers.
+In that case, be prepared to update your code if this module changes. This module may
+change between releases.
+"""
+
+from __future__ import annotations
+
+import json
+import shutil
+from io import StringIO
+
+
+import grass.script as gs
```

Comment:
isort+formatting gives:

```suggestion
from __future__ import annotations

import json
import shutil
from io import StringIO

import grass.script as gs
```